site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Allows controllers to be strictly for control loops; feature separates extensions and webhooks out of your controllers
```

Comment:
_I_ know what you mean here because I wrote language like this, but I would suggest making this more like: `Learn how to configure your Agones services for high availability.`